backend/controllers/caseNote.js

```diff
@@ -116,14 +116,14 @@ const caseNoteFactory = (elite2Api, caseNotesApi) => {
       })
     }
 
-    if ((hours && parseInt(hours, 10) > 23) || !hours) {
+    if ((hours && !(parseInt(hours, 10) <= 23)) || !hours) {
```

Comment:
parseInt returns NaN if letters are passed. NaN > a number and NaN less than a number both return `false`, switching how this checks the time value around means that it will behave correctly both in the case of a NaN and if the value is too high.